HHS/Head-Start-TTADP
# [PROD] Move filter from RTR list view to RTR search view. Update SSDI.

## Description of change

- The filter was put on the wrong view. This PR moves it to the RTR search, not the RTR goal list.
- on the delivery method graph on the QA dashboard: virtual data was showing up under in-person and vice-versa. Hybrid was correct.
- Monitoring Import Cadence adjustment to bypass minor: Through observation some days miss the window to get the latest monitoring data due to dns failures or system reboots. This change will trigger the collection 4 times per 24 hours in an attempt to recover before the next day should the intended window be missed. Nothing happens if it is triggered and no work needs to be done.
- Descriptor race condition: Without the changes if the same endpoint is hit for the first time more then once before the first transaction has completed, it will fail do to matching more then one descriptor id. The changes prevent this through a SQL function that would select only the first if multiple to exists, delete any dups that exists, and a constraint that prevents dups ever existing in the future.
- All roles: All roles need to be displayed to keep counts correct


## How to test
- Confirm (easiest way to do that is view the response from the network tab and comparing it to the graph) that the data on the graph is correct
- Validate the new cron schedule.
- Hit a page for the first time and no error will occur
- view the QA dashboard

## Issue(s)

* https://jira.acf.gov/browse/TTAHUB-3190
* https://jira.acf.gov/browse/TTAHUB-3586
* https://jira.acf.gov/browse/TTAHUB-3582
* https://jira.acf.gov/browse/TTAHUB-3585
* https://jira.acf.gov/browse/TTAHUB-3500

## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
